eo-maven-plugin/src/main/java/org/eolang/maven/SpyTrain.java

```diff
@@ -55,12 +57,16 @@ final class SpyTrain extends TrEnvelope {
                         shift::uid,
                         (pos, xml) -> {
                             final String log = shift.uid().replaceAll("[^a-z0-9]", "-");
-                            new Home().saveQuietly(
-                                xml.toString(),
-                                dir.resolve(
-                                    String.format("%02d-%s.xml", pos, log)
-                                )
-                            );
+                            try {
+                                new Home().save(
+                                    xml.toString(),
+                                    dir.resolve(
+                                        String.format("%02d-%s.xml", pos, log)
+                                    )
+                                );
+                            } catch (final IOException exc) {
+                                throw new RuntimeException(exc);
```

Comment:
@yasamprom Maybe we can use here `IllegalStateException` as it was in method `saveQuietly`?

Reply:
Hm, yes. Thanks!